tests/unit/sinon-sandbox-test.js

```diff
@@ -18,6 +18,23 @@ test('stores sandbox created as module property', function(assert) {
   restoreSandbox();
 });
 
+test('stubbing out sandbox.create returns the already created sandbox', function(assert) {
+  assert.expect(1);
+
+  createSandbox();
+
+  const sandbox =  QUnit.config.current.testEnvironment.sandbox;
+
+  const createdSandbox = sandbox.sandbox.create();
+
+  // nessessary to do a comparison, as the original sandbox has our extra `sandbox.create()`.
+  delete sandbox.sandbox;
```

Comment:
I don't think this should be needed, and it surprises me...

Reply:
Yep, there's a chicken and egg problem here. It surprised me too...

Reply:
I removed it. It worked. I must have been doing something dumb.